userspace/libsinsp/container.cpp

```diff
@@ -29,10 +29,24 @@ limitations under the License.
 #include "container.h"
 #include "utils.h"
 
+using namespace libsinsp;
+
 sinsp_container_manager::sinsp_container_manager(sinsp* inspector) :
 	m_inspector(inspector),
 	m_last_flush_time_ns(0)
 {
+#ifdef CYGWING_AGENT
+	m_container_engines.emplace_back(new container_engine::docker());
```

Comment:
Seems like you can add the docker resolver unconditionally.

Reply:
I was just following the logic of what the template methods did, but you're right that I can pull this out.